README.md

```
@@ -162,6 +164,10 @@ This table lists ALL environment variables used in the system. Variables without
 | `HLS_SEGMENT_DURATION`            | HLS segment duration in seconds                  | `4.0`                        | `4.0`                                                           | `conf/lib/00_settings.liq`             | All             |
 | `HLS_SEGMENTS`                    | Segments per live playlist                       | `10`                         | `10`                                                            | `conf/lib/00_settings.liq`             | All             |
 | `HLS_SEGMENTS_OVERHEAD`           | Extra old segments retained locally              | `5`                          | `5`                                                             | `conf/lib/00_settings.liq`             | All             |
+| **Stream Metadata (Optional)**    |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 6; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)